RFS/src/main/java/org/opensearch/migrations/bulkload/common/http/ConnectionContext.java

```diff
@@ -140,6 +143,9 @@ public static class TargetAdvancedArgs {
         @Parameter(names = {
             "--target-compression" }, description = "**Advanced**. Allow request compression to target", required = false)
         public boolean compressionEnabled = false;
+
+        @Parameter(names = { "--target-version-override" }, description = "**Advanced**. Allow overriding the version handling for the target cluster.", converter = VersionConverter.class)
```

Comment:
Before merging this, can we update the cdk to plum this to RFS workers since it already has the version key in the target_cluster object if specified

https://github.com/opensearch-project/opensearch-migrations/pull/960

Reply:
I'm backing off exposing the target-version override for now, there are code smell fixes and the inclusion of ES 6.X that are still worthwhile to move forward with.

Reply:
do we have a jira for the target version override so we don't lose this feature?

Reply:
Good call, created https://opensearch.atlassian.net/browse/MIGRATIONS-2101